Sources/TuistLoader/Models+ManifestMappers/Project+ManifestMapper.swift

```diff
@@ -67,11 +66,10 @@ extension TuistCore.Project {
                 targets: targets,
                 packages: packages,
                 schemes: schemes,
-                autogenerateSchemes: autogenerateSchemes,
                 additionalFiles: additionalFiles)
     }
 
-    func replacing(autogenerateSchemes: Bool) -> TuistCore.Project {
+    func replacing(autogenerateSchemes _: Bool) -> TuistCore.Project {
```

Comment:
I believe we can delete this entire method

Reply:
Good catch

Reply:
minor: this can still go :)